upload.sh

```diff
@@ -40,7 +44,7 @@ upload()
 # Upload default AWS ParallelCluster config file
 upload_config()
 {
-	aws s3 cp --acl public-read scripts/config.ini s3://${s3_bucket}/config.ini
+	aws s3 cp --acl public-read scripts/config.ini s3://${s3_bucket}/${version}/config.ini
```

Comment:
Is this still needed now that you added it as an asset?

Reply:
I think this should be kept. The asset has an obfuscated filename (uuid). This cp at least allows us to reference the versioned config.ini with a human-readable uri.